spec.html

```diff
@@ -5459,7 +5453,7 @@ <h1>IsInteger ( _argument_ )</h1>
 
     <emu-clause id="sec-isnonnegativeinteger" aoid="IsNonNegativeInteger">
       <h1>IsNonNegativeInteger ( _argument_ )</h1>
-      <p>The abstract operation IsNonNegativeInteger determines if _argument_ is non-negative integer Number value.</p>
+      <p>The abstract operation IsNonNegativeInteger takes argument _argument_. It determines if _argument_ is non-negative integer Number value. It performs the following steps:</p>
```

Comment:
```suggestion
      <p>The abstract operation IsNonNegativeInteger takes argument _argument_. It determines if _argument_ is a non-negative integral Number value. It performs the following steps:</p>
```

Reply:
Added the missing "a", but leaving this as "integer" rather than "integral" to match other operations. Feel free to submit a followup PR fixing all of them.